ownable-core/src/as_clone.rs

```diff
@@ -10,13 +10,15 @@ use core::ops::{Deref, DerefMut};
 /// Transparent wrapper for [`Clone`]able types to support all [traits](crate::traits) (by cloning).
 ///
 /// ```rust
-/// # use ownable::{AsClone, IntoOwned, ToBorrowed, ToOwned};
-/// # use std::borrow::Cow;
-/// # use std::collections::HashMap;
-/// // Simple struct which uses a `String` with the help of `AsClone`.
-/// // As an alternative it's possible to impl the ownable traits by hand (for own types, not `String`).
-/// #[derive(IntoOwned, ToBorrowed, ToOwned)]
-/// struct MyMap<'a>(HashMap<AsClone<String>, Cow<'a, str>>);
+/// use ownable_core::{AsClone, traits::IntoOwned};
+///
+/// #[derive(Clone)]
+/// struct NotIntoOwned;
+///
+/// fn requires_into_owned<O: IntoOwned>() {}
+///
+/// // `AsClone<NotIntoOwned>` implements `IntoOwned` through `Clone`
+/// requires_into_owned::<AsClone<NotIntoOwned>>();
```

Comment:
The examples for `AsCopy` and `AsClone` were using the derive macro, so I had to tweak them to not. I was going to add docs detailing their usage with the derive macros, but the docs for `ownable` already cover everything that I was going to